CHANGELOG.md

```diff
@@ -1,8 +1,35 @@
-# dbt_hubspot_source version.version
+# dbt_hubspot_source v0.18.0
+
+## Deprecations
+- Select fields have been deprecated from the `stg_hubspot__contact_list` model. The full removal date is planned for May 10th, 2025. Follow [Deprecation Issue #138](https://github.com/fivetran/dbt_hubspot_source/issues/138) for more details and updates around the planned sunsetting of the fields. ([#137](https://github.com/fivetran/dbt_hubspot_source/pull/137))
+  - The deprecated fields include the following:
+    - `is_deletable`
+    - `is_dynamic`
+    - `metadata_error`
+    - `metadata_last_processing_state_change_at`
+    - `metadata_last_size_change_at`
+    - `metadata_processing`
+    - `metadata_size`
+    - `portal_id`
+
+## Breaking Changes
+- Select fields have been added to the `stg_hubspot__contact_list` model. ([#137](https://github.com/fivetran/dbt_hubspot_source/pull/137))
+  - The deprecated fields include the following:
```

Comment:
I don't think these fields are being deprecated, [at least based on the initial issue](https://github.com/fivetran/dbt_hubspot/issues/155)--it wouldn't really make sense to add them otherwise. Would recommend

Reply:
Copy paste error. This was intended to callout the added fields.

I'd prefer to keep the bullets indented as they are since it looks a bit strange when there is a semi colon after a PR ref link.
```suggestion
  - The newly added fields include the following:
```